src/libcore/platforms/nodejs.js

```diff
@@ -381,11 +385,17 @@ export default (arg: {
               if (process.env.VERBOSE) {
                 log("libcore-call", id + "." + method, args);
               }
-              const constructorArgs =
-                typeof njsBuggyMethodIsNotStatic === "function"
-                  ? njsBuggyMethodIsNotStatic(args)
-                  : args;
-              const value = new m(...constructorArgs)[method](...args);
+              let value;
```

Comment:
mmh why is this now changing?

Reply:
Static doesn't seems to unwrap hex, so when i was sending hex string to libcore it wasn't converted to binary.
I couldn't find any example in types that needed to convert hex string in static